src/Polyester.jl

```diff
@@ -9,12 +9,12 @@ using Static
 using Requires
 using PolyesterWeave:
   request_threads, free_threads!, mask, UnsignedIteratorEarlyStop, assume,
-  disable_polyester_threads
-using CPUSummary: num_threads, num_cores
+  disable_polyester_threads,
+  num_threads # used to be taken from CPUSummary, but it caused significant TTFX | TODO remove on next breaking release
+using CPUSummary: num_cores
```

Comment:
`num_cores` should not be recompiling often (only if you move it between machines, or access it from a network file system), but we could consider dropping it.
It is used on line 323 of `src/closure.jl`, so users can optionally use the minimum of `min(nthreads(), num_cores()).

Reply:
Given that `num_cores` does not cause recompilation, I did not touch it. I added a comment to consider it at the next breaking release